dockerbuild/run.sh

```diff
@@ -2,4 +2,6 @@
 
 composer update
 
+cp ./config.php $SSP_PATH/config/config.php
```

Comment:
Where is $SSH_PATH defined?

Reply:
It's in the Dockerfile

# Copy in SSP override files
ENV SSP_PATH /data/vendor/simplesamlphp/simplesamlphp